build.gradle

```diff
@@ -144,3 +151,26 @@ uploadArchives {
 }
 
 apply from: 'changelog.gradle'
+
+dependencies {
+    testCompile mcTesterDep
+    testCompile "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    testCompile "org.jetbrains.kotlinx:kotlinx-coroutines-core:0.23.4"
+    testCompile "org.jetbrains.kotlin:kotlin-reflect:1.2.51"
+    testCompile "org.apache.logging.log4j:log4j-core:2.8.1"
+}
+
+kotlin {
+    experimental {
+        coroutines 'enable'
```

Comment:
Can you update this/McTester to Kotlin 1.3? As far as I know, coroutines are declared stable in 1.3, so we'd be able to remove this.

Reply:
Fixed